docs/docsite/rst/playbooks_tags.rst

```diff
@@ -73,7 +81,10 @@ Example::
 Tag Inheritance
 ```````````````
 
-You can apply tags to more than tasks, but they ONLY affect the tasks themselves. Applying tags anywhere else is just a convenience so you don't have to write it on every task::
+You can apply ``tags:`` to structures above tasks, but when Ansible processes
+them, ONLY the tasks they contain are tagged. Applying tags anywhere other than
+tasks is just a convenience so you don't have to tag all indivdual tasks. This
+example tags all tasks in a play::
```

Comment:
this is 'mostly' true, the exception are 'include_tasks/roles' which won't push the inheritance down

Reply:
Do you think that I need to say that earlier? You'll notice late in that block I do say 'The above information does not apply to `include_tasks`, `include_roles`,
    or other dynamic includes.', but I grant it could be said earlier somehow.

Reply:
i expect people to a) not read docs, b) read only till they think they got info they want, so 'early is better'

Reply:
Changed. See what you think.